src/xSTIR/cSTIR/include/sirf/STIR/stir_x.h

```diff
@@ -398,7 +437,8 @@ The actual algorithm is described in
 			sptr_asm_.reset();
 			//sptr_normalisation_.reset();
 		}
-		stir::shared_ptr<const PETAcquisitionModel> linear_acq_mod_sptr() const
+//		stir::shared_ptr<const PETAcquisitionModel> linear_acq_mod_sptr() const
```

Comment:
why do we need to get rid of the `const`? It seems to open us up for problems.

Reply:
`forward` and `backward` were not `const` - they are now

Reply:
(I was in a hurry to make this pull request before my annual leave next week, and was not sure imposing `const`-ness would go smoothly - it certainly will not on MR side - so went for quick fix)

Reply:
😄